package.json

```diff
@@ -27,21 +28,21 @@
   "homepage": "https://github.com/senecajs-labs/seneca-sqlite-store",
   "bugs": "https://github.com/senecajs-labs/seneca-sqlite-store/issues",
   "dependencies": {
-    "sqlite3":    ">=2.1.5",
-    "node-uuid":  "*",
     "eraro": "^0.4.1",
-    "lodash": "^4.3.0"
+    "lodash": "^4.3.0",
+    "node-uuid": "*",
+    "sqlite3": ">=2.1.5"
   },
   "devDependencies": {
-    "lab": "8.2.x",
-    "seneca": "plugin",
-    "seneca-store-test": "^1.0.0",
+    "@hapi/lab": "24.2.0",
     "eslint-config-seneca": "1.x.x",
+    "eslint-plugin-hapi": "4.x.x",
     "eslint-plugin-standard": "1.x.x",
-    "eslint-plugin-hapi": "4.x.x"
+    "seneca": "plugin",
+    "seneca-entity": "13.0.0",
+    "seneca-store-test": "^1.0.0"
```

Comment:
Q: Would it be okay to freeze these devDependencies?

Reply:
also yes